CMakeLists.txt

```diff
@@ -66,6 +67,18 @@ if (GLOW_WITH_OPENCL)
   find_package(OpenCL REQUIRED)
 endif ()
 
+# sets pre processor flag so some of glow projects
+# can link against protobuf libray that is build as DLL.
+if(LINK_PROTOBUF_AS_DLL)
+  # For protobuf warning when it is build as dll
+  # Supresses a warning that is treated as warning
+  # Basically one of the header files has interface class
+  # containing STL library. Which might cause issues
+  # if things are build with different compilers.
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4251")
```

Comment:
If we set LINK_PROTOBUF_AS_DLL to True on Linux things would not work. Can we wrap this in an if-def for windows?

Reply:
Added it as an AND.